5/README.md

```diff
@@ -45,23 +54,138 @@ There are many use cases for the usage of a Property:
 
 * To change the representation of an information in order to be more user friendly a Property can define a new data format for the graphical user interface to convert the raw data.
 
+* To configure some alarms on Attribute value.
 
-## Specification
-
-A Tango Property is a strict definition of a pair of key/value
-* configuration are represented in the form of properties.
-* data are represented in the form of attributes.
-* actions are represented in the form of commands.
-
-Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, Attribute, Command following the rules below:
+* To memorize the Attribute value each time it changes.
```

Comment:
Does this means "marking an attribute as memorized" ? I'd rewrite this as "To mark an attribute as memorized, and store the value each time it changes".

Reply:
@lorenzopivetta , I agree it is currently a bit ambiguous. I would not use "To mark an attribute as memorized" because this part is done in the code, not in the DB.
I would reformulate to something like:
* to memorize the Attribute value of a Memorized Attribute when it changes